src/index.html

```diff
@@ -4,9 +4,311 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Samsung</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Smartphones</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#"
+              class="navigation__link"
+              data-qa="nav-hover">
```

Comment:
Indentation should look like this 
```suggestion
              data-qa="nav-hover">
               <a 
                  href="#"
                  class="navigation__link"
                  data-qa="nav-hover"
               >
```